### ISSUE TITLE
Minute sheet/orders research

### ISSUE THREAD

**Issue by katiecissell** [ustaxcourt/ef-cms#8538]
As a Trial Clerk, so that I can enter data into minute sheets more efficiently, I need improvements made to the minute sheet pdfs.

As a court user, so that I can view information from a trial session without going into the minute sheet, I need that information to be displayed in a case.

## Notes
From interviews with trial clerks: https://app.mural.co/t/flexion6734/m/flexion6734/1646751586915/d083ccfb87868b6f9af6987fbb2895178f6b0bdb?sender=u043924e0f84e3ce8b4655599

Research questions:
Who is the person who uses the box on the right of the form? Is the box still relevant?
Who created that pdf? (the adobe person) Is that PDF still in use?
Who to interview/focus groups?

## Story Definition of Ready (updated on 12/23/22)
The following criteria must be met in order for the user story to be picked up by the Flexion development team.
The user story must: 
- [ ]  Is framed in business/user need, the value has been addressed.
- [ ] Includes acceptance criteria
- [ ] Has been refined
- [ ] Pre conditions have been satisfied. 
**Process:** 
Flexion developers and designers will test if the story meets acceptance criteria and test cases in Flexion dev and staging environments (“standard testing”). If additional acceptance criteria or testing scenarios are discovered while the story is in progress, a new story should be created, added to the backlog and prioritized by the product owner. 

## Definition of Done (Updated 5-19-22)
**Product Owner**
 - [ ]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios have been refined to meet all acceptance criteria 
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [ ] Automated test scripts have been written, including visual tests for newly added PDFs
 - [ ] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [ ] Verify that language for docket record for internal users and external users is identical
 - [ ] New screens have been added to pa11y scripts
 - [ ] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [ ] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [ ] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [ ] Interactors should validate entities before calling persistence methods
 - [ ] Code refactored for clarity and to remove any known technical debt
 - [ ] Acceptance criteria for the story has been met
 - [ ] If there are special instructions in order to deploy into the next environment, add them as a comment in the story
 - [ ] Reviewed by UX on a deployed environment.
 - [ ] Reviewed by PO on a deployed environment. Can be deployed to the Court's `test` environment if prod-like data is required. Otherwise deployed to any `experimental` environment.
 - [ ] Deployed to the Court's `staging` environment.

**Comment by liadmccabe** [comment]
Blocked: This story is currently blocked due to waiting on a date/time to meet with the Court for a focus group to discuss the non-trial minute sheets.

**Comment by swongCO** [comment]
Second focus group happened on 4/20. Feedback and insights collected in Mural board. Closing as done.
